libsyclinterface/tests/test_sycl_kernel_interface.cpp

```diff
@@ -145,14 +145,8 @@ TEST_P(TestDPCTLSyclKernelInterface, CheckGetPrivateMemSize)
     EXPECT_NO_FATAL_FAILURE(axpy_private_mem_sz =
                                 DPCTLKernel_GetPrivateMemSize(AxpyKRef));
 
-    if (DPCTLDevice_IsGPU(DRef)) {
-        ASSERT_TRUE(add_private_mem_sz != 0);
-        ASSERT_TRUE(axpy_private_mem_sz != 0);
-    }
-    else {
-        ASSERT_TRUE(add_private_mem_sz >= 0);
-        ASSERT_TRUE(axpy_private_mem_sz >= 0);
-    }
+    ASSERT_TRUE(add_private_mem_sz >= 0);
+    ASSERT_TRUE(axpy_private_mem_sz >= 0);
```

Comment:
This is actually an improvement in DPC++ RT, not a regression. Neither of the kernels use private memory.